config/application.rb

```diff
@@ -54,7 +48,7 @@ class Application < Rails::Application
       static-print.css
     }
 
-    config.assets.css_compressor = :yui
+    # config.assets.css_compressor = :yui
```

Comment:
Removed

I also:

- got rid of the `assets` Gem group
- pinned the asset gems as per the style guide
- removed the (now unused) yui gem